plugins/net_plugin/net_plugin.cpp

```diff
@@ -483,6 +484,11 @@ namespace eosio {
       mutable fc::mutex             chain_info_mtx; // protects chain_info_t
       chain_info_t                  chain_info GUARDED_BY(chain_info_mtx);
 
+      mutable fc::mutex                           seen_hs_msgs_mtx;
+      std::vector<std::unordered_set<fc::sha256>> seen_hs_msgs_buckets            GUARDED_BY(seen_hs_msgs_mtx);
+      uint64_t                                    seen_hs_msgs_current_bucket = 0 GUARDED_BY(seen_hs_msgs_mtx);
```

Comment:
This doesn't build with clang-16. 
```suggestion
      uint64_t                                    seen_hs_msgs_current_bucket     GUARDED_BY(seen_hs_msgs_mtx) = 0;
```